shared/packages.sh

```diff
@@ -96,6 +96,12 @@ OPUSFILE_URL="https://archive.mozilla.org/pub/opus/$lib-$ver.tar.gz"
 OPUSFILE_DIR="$lib-$ver"
 OPUSFILE_ARGS="--disable-http"
 
+lib=FluidLite
+ver=bce54560975bf1d69569314f46d027b06307ca1a
+FLUIDLITE_URL="https://github.com/divideconcept/$lib/archive/$ver.zip"
+FLUIDLITE_DIR="$lib-$ver"
+FLUIDLITE_ARGS="-DFLUIDLITE_BUILD_STATIC=ON -DFLUIDLITE_BUILD_STATIC=OFF"
```

Comment:
on or off? :P

Reply:
Oh. I meant shared xD

Reply:
Surprisingly this still compiled. Likely because Generic architecture is used that had no shared library support /*g/*